plugin/trino-google-sheets/src/test/java/io/trino/plugin/google/sheets/TestGoogleSheetsWithoutMetadataSheetId.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.google.sheets;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.testing.AbstractTestQueryFramework;
+import io.trino.testing.QueryRunner;
+import org.testng.annotations.Test;
+
+import static io.trino.plugin.google.sheets.SheetsQueryRunner.createSheetsQueryRunner;
+import static io.trino.plugin.google.sheets.TestSheetsPlugin.DATA_SHEET_ID;
+
+public class TestGoogleSheetsWithoutMetadataSheetId
```

Comment:
How long does it take to run this test class?

Reply:
Very fast, ~15 seconds on my local machine.

Reply:
what about adding a test case in TestSheetsPlugin (instead?)

Reply:
I do not think that helps, `TestSheetsPlugin` runs in 1-2 seconds currently. 

Moving the test case in there will just make `TestSheetsPlugin` take ~15 seconds instead of ~1-2 seconds.

Reply:
@grantatspothero i understand the approach currently taken is best from test coverage perspective.
@nineinchnick and I were recently debating our inherently limited ability to have test coverage, and how much we can test in total on the CI. Looks like there are and will always be limits to that. This made me more cautious about time being added to tests with respect to additional confidence gained when doing so. Ultimately, we need to accept the fact we cannot test everything, which is especially sad to me.

(no change requested here, but I'd still consider not having this test class)